techs/stylus.js

```diff
@@ -59,6 +10,84 @@ var path = require('path'),
     EOL = require('os').EOL,
     csswring = require('csswring');
 
+/**
+ * @class StylusTech
+ * @augments {BaseTech}
+ * @classdesc
+ *
+ * Build CSS from stylus and css sources.<br/><br/>
+ *
+ * Processing files into 3 steps:<br/>
+ * 1. Prepare list of @import sources (contain styl and css sources)<br/>
+ * 2. Expand stylus @import and process them of the [stylus renderer]{@link https://github.com/stylus/stylus}<br/>
+ * 3. Expand the remaining css @import and processes the received common css with help of
+ * [postcss]{@link https://github.com/postcss/postcss}<br/><br/>
+ *
+ * Important: options(prefix, includes, hoist, useNib) enable only for stylus sources<br/>
+ *
+ * @param {Object}          [options]                                 Options
+ * @param {String}          [options.filesTarget='?.files']           Path to target with
+ *                                                                    [FileList]{@link http://bit.ly/1GTUOj0}
+ * @param {String[]}        [options.sourceSuffixes=['styl', 'css']]  Files with specified suffixes involved in
+ *                                                                    the assembly. Default: `styl`, `css`
+ * @param {String|Boolean}  [options.url='rebase']                    Rebase or inline url():<br/>
+ *                                                                    - `rebase` resolves a path relative to the bundle
+ *                                                                    directory;<br/>
+ *                                                                    - `inline` inline assets using base64 encoding.
+ * @param {Boolean}         [options.comments=true]                   Add css comments with path to source
```

Comment:
Add —AddS
css —> CSS
Adds CSS comment with path to source to a code block (above and below).